installer.sh

```diff
@@ -161,9 +161,9 @@ completed() {
 check_java_version() {
   java_version=$(java -version 2>&1 | awk -F '"' '/version/ {print $2}')
   echo "Your current java version is ${java_version}"
-  # Split the Java version into parts and check if the major version is 17 or higher
-  IFS=. read -ra version_parts <<< "$java_version"
-  if [ "${version_parts[0]}" -ge 17 ]; then
+  # Split the Java version string into parts based on the '.' delimiter
+  major_version=$(echo "$java_version" | cut -d'.' -f1)
+  if [ "$major_version" -ge 17 ]; then
```

Comment:
did you test it after on other operating systems to see if won't impact them?

Reply:
I had ubuntu vm installed on windows, it is working fine on that. Also tested on fish shell - https://fishshell.com/
working fine.

Also the error "<<<" breaks in dash shell, the current code works in dash shell